buttons.lua

```diff
@@ -90,37 +95,44 @@ function MaxDps:ApplyOverlayChanges()
 	end
 end
 
-function MaxDps:UpdateButtonGlow()
-	local LAB;
-	local LBG;
+do
```

Comment:
Why is this `do` needed?

Reply:
To isolate `origShow` which is related to `UpdateButtonGlow` there's no need to expose it to the rest of the code. Pseudo-encapsulation, sorta.